packages/thirdweb/package.json

```diff
@@ -233,7 +233,10 @@
     "fuse.js": "7.1.0",
     "input-otp": "^1.4.1",
     "mipd": "0.0.7",
+    "open": "^10.1.0",
+    "ora": "^8.2.0",
     "ox": "0.6.10",
+    "prompts": "^2.4.2",
```

Comment:
can you use static versions for dependencies please, especially for the hard dependencies